fedgov-cv-ISDcreator-webapp/src/main/webapp/js/features.js

```diff
@@ -337,8 +337,8 @@ function placeComputedLane(newDotFeature, lanes, vectors, laneMarkers, laneWidth
  * Function to build a computed feature.
  * @param {*} i Lane identifier
  * @param {*} laneNumber Lane number 
- * @param {*} referenceLaneID Reference lane ID
- * @param {*} referenceLaneNumber  Reference lane number
+ * @param {*} referenceLaneID Reference lane Number
```

Comment:
if the description is different, should the variable name be different to avoid confusion?

Reply:
https://github.com/usdot-fhwa-stol/connectedvcs-tools/pull/90/commits/9a91fcc04a19229453165a1537df75575e052635